zenoss_hipchat/hipchat.py

```diff
@@ -88,27 +88,34 @@ def _should_notify(self):
         Determine if we should set the notify flag in HipChat call
         """
         if self.severity >= self.NOTIFY_SEVERITY:
-            return 1
-        return 0
+            return True
+        return False
 
     def send(self):
         """
         Send off the message to environment room with a nice template
         """
-
         if not self.clear:
             message = self._event_message()
         else:
             message = self._clear_message()
 
-        from_name = "{0} ({1})".format(
+        if config.HIPCHAT_FROM == '':
+            from_name = "{0}".format(
+            self.SEVERITY_MAP[self.severity][1]
+        )
+        else:
+            from_name = "{0} ({1})".format(
+            config.HIPCHAT_FROM, self.SEVERITY_MAP[self.severity][1]
+        )
+        
+        from_name = "{1}".format(
             config.HIPCHAT_FROM, self.SEVERITY_MAP[self.severity][1]
```

Comment:
I think you want to remove the two lines above or they will overwrite everything done in the if/else above